src/app/community/community.component.ts

```diff
@@ -51,7 +55,10 @@ export class CommunityComponent implements OnInit {
   }
 
   ngOnInit() {
+    var urlFragment = this.router.url.split('/')[2];
+    if(urlFragment != "" && urlFragment != undefined){
```

Comment:
You could shorten it to if (!!urlFragment) { since it's a truthy value. More info: https://basarat.gitbooks.io/typescript/docs/tips/truthy.html

Reply:
@LKhadka That's really useful!
@Pr0chin You should also use `!==` rather than `!=`.  There is a lint rule for this, so please make sure to run the `ng lint` either with the pre push hook or from your terminal and fix the errors it finds.

Reply:
ok thanks for your feedback @LKhadka @paulbert

Reply:
@LKhadka  if(!! variable) is not a feasible condition we can imply if(! variable) for truth values.
![screeen](https://user-images.githubusercontent.com/24644003/32820591-5041cd46-c9f7-11e7-8dfe-b90df93cab6e.png)